web/pdf_sidebar.js

```diff
@@ -20,6 +20,7 @@ import {
   toggleCheckedBtn,
   toggleExpandedBtn,
 } from "./ui_utils.js";
+import { NullL10n } from "display-l10n_utils";
```

Comment:
I just realized that the `PDFSidebar` is *always assumed* to be initialized with a "proper" l10n-class, since it's not part of the standalone viewer-components [as defined here](https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.component.js).

Hence you can actually remove all `NullL10n` stuff in this particular file; sorry about not catching this earlier!